man/admixMap.Rd

```diff
@@ -7,14 +7,16 @@ Run admixture analyses
 }
 \usage{
 admixMap(admixDataList, null.model, male.diploid=TRUE,
-         genome.build=c("hg19", "hg38"), verbose=TRUE)
+         genome.build=c("hg19", "hg38"), 
+         BPPARAM=bpparam(), verbose=TRUE)
 }
 
 \arguments{
   \item{admixDataList}{named list of \code{\link{GenotypeIterator}} or \code{\link{SeqVarIterator}} objects for each ancestry}
     \item{null.model}{A null model object returned by \code{\link{fitNullModel}}.}
   \item{male.diploid}{Logical for whether males on sex chromosomes are coded as diploid.}
   \item{genome.build}{A character sting indicating genome build; used to identify pseudoautosomal regions on the X and Y chromosomes.}
+  \item{BPPARAM}{A \code{\link{BiocParallelParam}} object to process blocks of variants in parallel.}
```

Comment:
should we say something about what the default does here?

Reply:
I'm hesitant to give too much detail in case BiocParallel changes how the `bpparam` function behaves, but I included a link to the `bpparam` documentation. (Notably, it does something different depending on the OS, as Windows doesn't support Multicore.)